### ISSUE TITLE
Send page: All button should fill the amount with a number

### ISSUE THREAD

**Issue by rating89us** [monero-project/monero-gui#2466]
![image](https://user-images.githubusercontent.com/45968869/69486097-44a01f00-0e48-11ea-876e-9b84b56e2f40.png)

When user clicks in All button, it should just fill the amount value in the amount field (in XMR), and not display "(all)". 

(all) is not specific. What is (all)? All balance in the wallet? In the account?

**Comment by aids9345435345kek** [comment]
> ![image](https://user-images.githubusercontent.com/45968869/69486097-44a01f00-0e48-11ea-876e-9b84b56e2f40.png)
> 
> Displaying a text "(all)" isn't specific. The user may think "What is (all)? All balance in the wallet? All unlocked balance? All balance in the account?"
> 
> Solution: when user clicks All button, it should just fill the amount field with total balance value (in XMR).

I agree with another user that replicating wasabi wallet functionality is desirable i.e the 'all' amount would be all minus the fee and would change dynamically as the user modifies the fee drop-down.

**Comment by rating89us** [comment]
> the 'all' amount would be all minus the fee

And below amount field we could add a text like this:
_+ network fee of 0.000037 XMR (0.01 USD)_
